feedhenry/fh-messaging
# task(CVE-2017-18214): RHMAP-20716 - Upgrade moment dependency

## JIRA
https://issues.jboss.org/browse/RHMAP-20716

## WHAT
- Upgrade moment dependency from `~2.15.2` to `~2.22.2` 

## WHY
Solve vulnerability. For further information check: https://nvd.nist.gov/vuln/detail/CVE-2017-18214

## USAGE
Used to manipulate dates 
PS.: It is not a break change/update. See the moment [Changelog](https://github.com/moment/moment/blob/develop/CHANGELOG.md)

## STEPS TO TEST
Run tests, check if the checks in PR was succeeded. The tests are using the moment and covering this points, however, it is not a breaking change at all.
